components/PlaylistTracks.tsx

```diff
@@ -1,24 +1,13 @@
-import { Artist, SpotifyAlbum } from '@/models/Spotify';
+import { PlaylistTrack } from '@/models/Spotify';
```

Comment:
I didn't notice this previously, but why can't we just use the `TrackCard` component for this rather than having this separate component?

Reply:
changed to use track component

Reply:
@Philip-Loeffler  Perfect, we should be able to remove this component then